src/Features/Core/Portable/CodeFixes/AbstractTopLevelConfigurationOrSuppressionCodeAction.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    internal abstract class AbstractTopLevelConfigurationOrSuppressionCodeAction : CodeAction.CodeActionWithNestedActions
+    {
+        protected AbstractTopLevelConfigurationOrSuppressionCodeAction(ImmutableArray<CodeAction> nestedActions, string title)
+            : base(title, nestedActions, isInlinable: false)
+        {
+        }
+
+        // Put configurations/suppressions at the end of everything.
+        internal override CodeActionPriority Priority => CodeActionPriority.None;
```

Comment:
interesting.  is that where 'None' goes?  it's lower than `.Low`?  Or does this mean: the  codefix infrastructure will determine where this goes.

Reply:
The suppression code fix already had None priority [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/CodeFixes/Suppression/TopLevelSuppressionCodeAction.cs,20), and the ordering logic is already ensuring that these go at the very bottom of the lightbulb.